components/ai_chat_ui/components/input_box/index.tsx

```diff
@@ -64,7 +66,10 @@ function InputBox (props: InputBoxProps) {
   return (
     <div className={styles.container}>
       {summaryFailedView}
+      {props.showSummarizeButton &&
```

Comment:
We cache the summarized text inside the TabHelper to save costs when the user clicks the button twice. But if it's a single-shot process, saving it in-memory may not be necessary. 

What do you think?

Reply:
Not sure, separate consideration.